data/adiabatic-conditions/ascii-data/isentrope_properties.txt

```diff
@@ -0,0 +1,504 @@
+# POINTS: 501 
```

Comment:
There can presumably be many isentropic adiabatic profiles, depending on which material model is used. As a consequence, would it make sense to reflect this in the name of the file? E.g., Burch-Monaghan-isentrope-properties.txt or something?